hawc/apps/summary/templates/summary/datapivot_form.html

```diff
@@ -13,7 +13,7 @@
   {{ smart_tag_form.media }}
   <script type="text/javascript">
     $(document).ready(function () {
-      let js_units_choices = {{form.js_units_choices|safe}},
+      let js_units_choices = {% autoescape off %}{% firstof form.js_units_choices "[]" %}{% endautoescape %},
```

Comment:
would this work instead? A little less noisy w/ the autoescape on/off.

`{% firstof form.js_units_choices|safe "[]" %}`

https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#firstof

Reply:
Actually, cancel that. After testing it doesn't work. The way you did actually works!